### PR TITLE
Fix Layout with Filament Grid Style and HasColumns Trait

### PR DESCRIPTION
**Pull request by slimani-dev**
### Overview
This update addresses layout issues in the developer logins component by implementing Filament's grid system and the HasColumns trait. These changes ensure the layout displays correctly across different screen sizes and provides better flexibility.

### Details
- **Grid Layout**: Replaced the previous layout setup with Filament's grid system, which ensures consistent and correct display across screen sizes.
- **HasColumns Trait**: Added the HasColumns trait for easier column management, allowing for customizable layouts and improved responsiveness.
- **Documentation Update**: Enhanced the README with a detailed guide on using the columns() method to adjust the grid layout.

### Why this change?
The previous layout was not displaying correctly, causing inconsistencies in the design. By switching to Filament's grid system and HasColumns trait, the layout now displays as intended across various devices and screen sizes.

### Testing
These changes were verified locally, focusing on different screen sizes to ensure the layout displays as expected and remains consistent.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,18 +39,46 @@ In the `users` method you can define the users (note: the users must exist), the
...
 the enabled() method also accepts a closure if you wish to enable the plugin based on a custom condition.
 
 Example:
 
 ```php
 // ...
 FilamentDeveloperLoginsPlugin::make()
-    ->enabled(app()->environment('local'))
+    ->columns() // default 2
 ```
```
Comment (bramr94) [inline README.md:68]: The documentation is not correct, some of the columns documentation is added under the enabled heading.

File: resources/views/components/developer-logins.blade.php [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
+                :sm="$columns['sm'] ?? null"
+                :md="$columns['md'] ?? null"
+                :lg="$columns['lg'] ?? ($columns ? (is_array($columns) ? null : $columns) : 2)"
+                :xl="$columns['xl'] ?? null"
+                :two-xl="$columns['2xl'] ?? null"
+                :attributes="\Filament\Support\prepare_inherited_attributes($attributes)->class('fi-wi gap-4')"
+        >
+            @foreach ($users as $label => $credentials)
+                @if ($loop->first)
+                @endif
```
Comment (bramr94) [inline resources/views/components/developer-logins.blade.php:31]: This can be removed.

File: src/FilamentDevelopersLogin.php [hunk 3]
```diff
@@ -19,6 +19,7 @@ public function login(Panel $panel, FilamentDeveloperLoginsPlugin $plugin, strin
 
         if ($panel->auth()->check()) {
             $panel->auth()->logout();
+            session()->forget('password_hash_'.$panel->getAuthGuard());
```
Comment (bramr94) [inline src/FilamentDevelopersLogin.php:22]: Is this really what caused the route not found bug? Can you explain why this fixes it?

**Review by bramr94 — COMMENTED** [review]
Can you also fix the merge conflicts?

**Comment by slimani-dev** [comment]
fixed all the problems with the merge conflict and the docs

**Pull request merged**